reference/docs-conceptual/overview.md

```diff
@@ -1,206 +1,89 @@
-ToString     Method     string ToString()
-Drive        Property   System.Management.Automation.PSDriveInfo Drive {get;}
-Path         Property   string Path {get;}
-Provider     Property   System.Management.Automation.ProviderInfo Provider {get;}
-ProviderPath Property   string ProviderPath {get;}
-```
-
-`Get-Location` returns a **PathInfo** object that contains the current path and other information.
-
-## Built-in help system
-
-Similar to Unix `man` pages, PowerShell includes detailed help articles that explain PowerShell
-concepts and command syntax. Use the [Get-Help][] cmdlet to display these articles at the command
-prompt or view the most recently updated versions of these articles in the PowerShell documentation
-online.
+PowerShell is a cross-platform task automation solution made up of a command-line shell, a scripting
+language, and a configuration management framework. PowerShell runs on Windows, Linux, and macOS.
+
+## Shell
+
+PowerShell is modern command shell that includes the best features of other popular shells. Unlike
+most shells that only accept and return text, PowerShell accepts and returns .NET Objects. The shell
+includes the following features:
+
+- Robust command-line [history][]
+- Tab completion and command prediction (See [about_PSReadLine][])
+- Supports command and parameter [aliases][]
+- [Pipeline][] for chaining commands
+- In-console [help][] system, similar to Unix `man` pages
+
+## Scripting language
+
+PowerShell is built on the .NET Common Language Runtime (CLR). All inputs and outputs are .NET
+objects. No need to parse text output to extract information from output. The PowerShell scripting
+language includes the following features:
+
+- Extensible through [functions][], [classes][], [scripts][], and [modules][]
+- Extensible [formatting system][formatting] for easy output
+- Extensible [type system][types] for creating dynamic types
+- Built-in support for common data formats like [CSV][], [JSON][], and [XML][]
+
+## Configuration management
+
+PowerShell Desired State Configuration ([DSC][]) is a management framework in PowerShell that
+enables you to manage your enterprise infrastructure with configuration as code. With DSC, you can:
+
+- Create declarative [configurations][] and custom scripts for repeatable deployments
+- Enforce configuration settings and report on configuration drift
+- Deploy configuration using [push or pull][push-pull] models
 
 ## Next steps
 
-To learn more about PowerShell, see the **Learning PowerShell** section of this site.
+### Getting started
+
+Are you new to PowerShell and don't know where to start? Take a look at these resources.
+
+- [Installing PowerShell][install]
+- [PowerShell 101][PS101]
+- [PowerShell Bits tutorials][tutorials]
+- [PowerShell Learn modules][learn]
+
+### PowerShell in action
+
+Take a look at how PowerShell is being used in different scenarios and on different platforms.
+
+- [Getting started with Azure PowerShell][azure]
```

Comment:
Can we add a PowerShell remoting one here as well for remote management of systems?

Reply:
done